MyMentorProject/src/main/java/com/Deny/MyMentorProject/Services/UserService.java

```diff
@@ -0,0 +1,51 @@
+package com.Deny.MyMentorProject.Services;
+
+import com.Deny.MyMentorProject.User.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    User user;
+
+
+  public UserService(User user) {       this.user = user;
```

Comment:
User is a model class that frequently changes so it should not be injected into this class. Instead you would want it to be an input to a method.